RELEASES.md

```diff
@@ -1,3 +1,211 @@
+Version 1.26.0 (2018-05-10)
+==========================
+
+Language
+--------
+- [Closures now implement `Copy` and/or `Clone` if all captured variables
+  implement either or both traits.][49299]
+- [The inclusive range syntax e.g. `for x in 0..=10` is now stable.][47813]
+- [Stablise `'_`. The underscore lifetime can be used anywhere where a
+  lifetime can be elided.][49458]
+- [`impl Trait` is now stable allowing you to have abstract
+  return types.][49255] e.g. `fn foo() -> impl Iterator<Item=u8>` or
+  `fn open(path: impl AsRef<Path>)`.
+- [Pattern matching on references are now automatically dereferenced.][49394]
```

Comment:
This sentence reads oddly, but it might just be me -- could we reword to "Pattern matching will now automatically apply dereferences" or something like that?